tests/unittests/test_transforms.py

```diff
@@ -959,3 +959,26 @@ def test_double_4(self):
     def test_no_double(self):
         t = plus_one >> plus_one / batch >> batch / plus_one >> plus_one
         t.pd_forward
+
+
+class TestTrace:
+    @pytest.fixture(autouse=True, scope='class')
+    def init(self, request):
+        emb = transform(torch.nn.Embedding)(10, 8)
+        linear = transform(torch.nn.Linear)(8, 4)
+        to_tensor = transform(lambda x: torch.LongTensor(x))
+        request.cls.pipeline = to_tensor >> emb >> linear
+
+    def test_pd_trace(self):
+        try:
+            list(self.pipeline.train_apply([[9, 8, 8], [4, 4, 4], [5, 5, 5], [6, 6, 6]],
+                                           batch_size=2, num_workers=0))
```

Comment:
It doesn't look like this `pipeline` produces an `RuntimeError` exception. I think the linear layer above needs to be changed to something like `linear = transform(torch.nn.Linear)(5, 4)`

Reply:
right, missed it